packages/documentation/copy/en/javascript/JSDoc Reference.md

```diff
@@ -240,6 +221,28 @@ export const userAccount = {
 var x = require("./accounts").userAccount;
 ```
 
+### `@import`
+
+`@import` can be used for type-only imports:
```

Comment:
```suggestion
The `@import` tag can lets us reference exports from other files.
```

Reply:
I'd also add a note about the targeted version:
```suggestion
As of [TypeScript 5.5](/docs/handbook/release-notes/typescript-5-5.html#the-jsdoc-import-tag),  the `@import` tag lets us reference exports from other files.
```